Userland/Libraries/LibWeb/CSS/CSSStyleSheet.cpp

```diff
@@ -99,12 +99,27 @@ WebIDL::ExceptionOr<void> CSSStyleSheet::remove_rule(unsigned index)
 
 void CSSStyleSheet::for_each_effective_style_rule(Function<void(CSSStyleRule const&)> const& callback) const
 {
+    for (auto& media : media()) {
+        if (!media.matches()) {
+            return;
```

Comment:
Is this the correct behavior? What if my style sheet has `media="print, screen"`? We will not match `print` but we have to keep going so we see `screen`.